pylint/checkers/spelling.py

```diff
@@ -42,7 +42,19 @@
     )
 except ImportError:
     enchant = None
-    # pylint: disable=no-init
+
+    class EmailFilter:  # type: ignore
+        ...
```

Comment:
Just my option, but I would use `pass` instead. Although equivalent, I associate `...` with stub files.

Reply:
I think this is a valid concern but seeing the pipeline are so long I'll probably merge as it is if they're is no other remarks.

Reply:
I just think that if you merge it now, it will probably stay that way forever. Yea, Travis might take some time, two builds are in the queue already, but it might be worth it.

Could make sense to combine it with a rebase. I noticed that the MR includes the pre-commit updates that you merged with #4077

Reply:
Yes, this is going to disappear when rebasing + merging.